examples/rename_brainvision_files.py

```diff
@@ -30,36 +30,50 @@
 
 ###############################################################################
 # We are importing everything we need for this example:
-import os
+import os.path as op
+import subprocess
 
 from numpy.testing import assert_array_equal
 from mne.io import read_raw_brainvision
 
-from mne_bids.datasets import fetch_brainvision_testing_data
 from mne_bids.copyfiles import copyfile_brainvision
 
 ###############################################################################
 # Step 1: Download some example data
 # ----------------------------------
 # To demonstrate the MNE-BIDS functions, we need some testing data. Here, we
-# will use data that is shipped with MNE-Python. Feel free to use your own
-# BrainVision data.
-examples_dir = fetch_brainvision_testing_data()
+# will use the AWS cli to download some BrainVision data. Feel free to use your
+# own BrainVision data.
+
+# First specify, where we want to download our data to
+examples_dir = op.join(op.expanduser('~'), 'mne_data', 'mne_bids_examples')
+
+# Now specify the data in the S3 remote storage and download the files
+data_address = 's3://openneuro.org/ds001810/'
+remote_dir = 'sub-01/ses-anodalpre/eeg/'
+fname = 'sub-01_ses-anodalpre_task-attentionalblink_eeg'
+for extension in ['.vhdr', '.vmrk', '.eeg']:
+    remote_file = data_address + remote_dir + fname + extension
+    cmd = ['aws', 's3', 'cp', '--no-sign-request', remote_file, examples_dir]
+    subprocess.run(cmd)
```

Comment:
reconsidering: I actually think I could just use `mne.datasets.testing.data_path()` to get testing BrainVision files 

... then I don't need to use `awscli`

Reply:
opinions @agramfort @jasmainak ?